web/src/app/shared/components/tree/tree.component.spec.ts

```diff
@@ -103,15 +115,25 @@ describe('TaskanaTreeComponent', () => {
     expect(classification.parentId).toEqual('id3');
     expect(classification.parentKey).toEqual('key3');
     expect(storeSpy.dispatch).toHaveBeenCalledWith(new UpdateClassification(classification));
-    expect(component.switchTaskanaSpinner).toHaveBeenCalledWith(true);
-    expect(component.switchTaskanaSpinner).toHaveBeenCalledWith(false);
```

Comment:
Since I've put the switchTaskanaSpinner(false) statement within the inner subscription I have no idea to test this..